packages/astro/src/cli/add/index.ts

```diff
@@ -87,7 +92,9 @@ async function getRegistry(): Promise<string> {
 }
 
 export async function add(names: string[], { flags }: AddOptions) {
-	telemetry.record(eventCliSession('add'));
+	const inlineConfig = flagsToAstroInlineConfig(flags);
+	const { userConfig } = await resolveConfig(inlineConfig, 'add');
+	telemetry.record(eventCliSession('add', userConfig));
```

Comment:
Is the current config actually important to record here?

Reply:
I don't have a problem if this info is statistically useful, but it seems like we'll get final data when users run `dev` or the `build` so would err on the side of not recording this.

Reply:
Maybe we can pass an empty user config object instead so we don't have to load the entire thing? e.g. `telemetry.record(eventCliSession('add', {}))`

Reply:
Is the reason that we don't track it here because it's slow? 

I see approvals so I'll merge but happy to revisit this post-PR if there's still interest in removing!